package.json

```diff
@@ -3,15 +3,16 @@
   "description": "A library for writing modern OpenTripPlanner-compatible multimodal journey planning web applications using React and Redux",
   "main": "build/index.js",
   "scripts": {
-    "build": "mastarm build",
+    "build": "mastarm build --env production",
     "cover": "mastarm test --coverage",
     "jest": "mastarm test",
     "lint": "mastarm lint lib --quiet",
     "lint-docs": "documentation lint lib/**/*.js",
     "prepublish": "mastarm prepublish --config configurations/prepublish",
     "prestart": "yarn",
     "test": "yarn run lint && yarn run lint-docs && yarn run jest",
-    "start": "mastarm build --serve example.js"
+    "start": "mastarm build --serve example.js",
+    "zip": "zip -r dist.zip index.html example.css dist"
```

Comment:
I'm pretty sure the zip command won't work on windows. What is the purpose of this script?

Reply:
We can remove this. I just wanted to preserve how to deploy the example. Thoughts on an alternative approach?